diff -purN a/src/main/java/com/lowagie/text/pdf/PdfGraphics2D.java b/src/main/java/com/lowagie/text/pdf/PdfGraphics2D.java
--- a/src/main/java/com/lowagie/text/pdf/PdfGraphics2D.java	2017-01-29 00:33:47.000000000 +0000
+++ b/src/main/java/com/lowagie/text/pdf/PdfGraphics2D.java	2017-02-02 14:57:42.027253886 +0000
@@ -186,10 +186,18 @@ public class PdfGraphics2D extends Graph
     }
     
     /**
+     * Shortcut constructor for PDFGraphics2D.
+     *
+     */
+    public PdfGraphics2D(PdfContentByte cb, float width, float height) {
+        this(cb, width, height, null, false, false, 0);
+    }
+    
+    /**
      * Constructor for PDFGraphics2D.
      *
      */
-    PdfGraphics2D(PdfContentByte cb, float width, float height, FontMapper fontMapper, boolean onlyShapes, boolean convertImagesToJPEG, float quality) {
+    public PdfGraphics2D(PdfContentByte cb, float width, float height, FontMapper fontMapper, boolean onlyShapes, boolean convertImagesToJPEG, float quality) {
         super();
         dg2.setRenderingHint(RenderingHints.KEY_FRACTIONALMETRICS, RenderingHints.VALUE_FRACTIONALMETRICS_ON);
         setRenderingHint(RenderingHints.KEY_FRACTIONALMETRICS, RenderingHints.VALUE_FRACTIONALMETRICS_ON);
